connect/transforms/src/main/java/org/apache/kafka/connect/transforms/ReplaceField.java

```diff
@@ -78,6 +79,10 @@ interface ConfigName {
             .define(ConfigName.REPLACE_NULL_WITH_DEFAULT_CONFIG, ConfigDef.Type.BOOLEAN, true, ConfigDef.Importance.MEDIUM,
                     "Whether to replace fields that have a default value and that are null to the default value. When set to true, the default value is used, otherwise null is used.");
 
+    private static Set<String> names() {
```

Comment:
nit: I think it is more clearly to use `configNames`

Reply:
Done. Thanks for review.